checker/tests/ainfer-resourceleak/non-annotated/MustCallAliasOnReceiver.java

```diff
@@ -0,0 +1,29 @@
+import java.io.*;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.mustcall.qual.*;
+
+@InheritableMustCall("close")
+public class MustCallAliasOnReceiver {
```

Comment:
Right now we don't have support adding `@MustCallAlias` annotation to the receiver parameter, so I removed the related test from this file to avoid crashes, I added it back, and instead I added `// @skip-test` in this file.